src/core/reducers/autocomplete.js

```diff
@@ -0,0 +1,64 @@
+export const AUTOCOMPLETE_LOADED = 'AUTOCOMPLETE_LOADED';
+export const AUTOCOMPLETE_STARTED = 'AUTOCOMPLETE_STARTED';
+export const AUTOCOMPLETE_CANCELLED = 'AUTOCOMPLETE_CANCELLED';
+
+const initialState = {
+  loading: false,
+  suggestions: [],
```

Comment:
As you mentioned, since autocomplete is a secondary feature, it's probably fine to default it to an empty array. Just keep this pattern in mind for future reducers: this approach will never allow the app to tell if an initial API request has been made or not (which might pose a problem).